### ISSUE TITLE
Delete the __main__ in twisted/names/root.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3712]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#3712|
|Type|enhancement|
|Created|2009-03-30 17:05:10Z|
|Branch|https://github.com/twisted/twisted/tree/root-main-3712|

This file isn't meant to be used as a script.  There are also no tests for this behavior.  Also it's using a soon-to-be-deprecated API.  We can add a real script to do name resolution with twisted.names later, if we want.

Attachments:

* [root.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/abc/abc97ffff99e82ffaa5729902f0fac3f95bc6ab1/27426c38dd5cd14fe6af6e2fb45adb35f52cd00d.patch) (718 bytes) - added by cary on 2009-03-30 21:49:18Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__3712 3712
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_root_main_3712 branches/root-main-3712
branch_author__thijs___cary thijs,  cary
status__closed closed
resolution__fixed fixed
component__names names
keywords__easy easy
time__1238432710000000 1238432710000000
changetime__1238473557000000 1238473557000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="cary's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|cary removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

(In [26552]) Branching to 'root-main-3712'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

(In [26553]) Apply root.patch, refs [#3712](https://github.com/twisted/twisted/issues/3712)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set owner to @exarkun|
|-|-|

Looks good.  I'm going to make a couple minor coding standard updates to the changed file.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun commented|
|-|-|

(In [26565]) Update copyright date, remove unused imports

refs [#3712](https://github.com/twisted/twisted/issues/3712)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_8"></a>|@exarkun set status to `closed`|
|-|-|

(In [26569]) Merge root-main-3712

Author: cary
Reviewer: thijs, exarkun
Fixes: [#3712](https://github.com/twisted/twisted/issues/3712)

Remove the `__main__` from `twisted/names/root.py`.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
